src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ConversionCompletionProviderTests.cs

```diff
@@ -1115,5 +1115,30 @@ public void Main()
 ", "int", displayTextSuffix: ")", expectedDescriptionOrNull: @"E.explicit operator int(E value)
 Defines an explicit conversion of a E to a int.");
         }
+
+        [WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(47511, "https://github.com/dotnet/roslyn/issues/47511")]
+        [InlineData("e.$$", true)]
+        [InlineData("e. $$", true)]
+        [InlineData("e.in$$", true)]
+        [InlineData("E.$$", false)] // Don't infer with enum member suggestion 
+        [InlineData("E.One.$$", true)]
+        public async Task ExplicitBuildInEnumConversionToIntAreOffered(string expression, bool conversionIsOffered)
+        {
+            Func<string, Task> verifyFunc = conversionIsOffered
+                ? markup => VerifyItemExistsAsync(markup, "int", displayTextSuffix: ")")
```

Comment:
quesiton: why does this validate the suffix, but not hte prefix?

Reply:
I think it is the first time `displayTextPrefix` is ever used. So support for it is very limited. I was reluctant to change `VerifyItemExistsAsync` because it uses default parameters and so I may introduce breaking changes if I place the parameter before `displayTextSuffix`. Should I add it anyway and if should I add it at the end of the parameter list or before `displayTextPrefix`?
```CS
        private protected async Task VerifyItemExistsAsync(
            string markup, string expectedItem, string expectedDescriptionOrNull = null,
            SourceCodeKind? sourceCodeKind = null, bool usePreviousCharAsTrigger = false,
            int? glyph = null, int? matchPriority = null, bool? hasSuggestionModeItem = null,
            string displayTextSuffix = null, string inlineDescription = null,
            List<CompletionFilter> matchingFilters = null, CompletionItemFlags? flags = null)
```

Reply:
I added support for displayTextPrefix in 305b723 and 2 follow up commits. I needed to touch a couple of other test files to do a proper integration.

Reply:
sgtm.